extensions/machine-learning-services/package.json

```diff
@@ -27,28 +28,66 @@
   "contributes": {
     "commands": [
       {
-        "command": "ml.command.managePackages",
-        "title": "%ml.command.managePackages%"
+        "command": "mls.command.managePackages",
+        "title": "%mls.command.managePackages%"
+      },
+      {
+        "command": "mls.command.odbcdriver",
+        "title": "%mls.command.odbcdriver%"
+      },
+      {
+        "command": "mls.command.mlsdocs",
+        "title": "%mls.command.mlsdocs%"
+      }
+    ],
+    "dashboard.tabs": [
+      {
+        "id": "ml-server",
+        "description": "Machine Learning Services",
+        "provider": "MSSQL",
+        "title": "Machine Learning Services",
+        "when": "connectionProvider == 'MSSQL' && !mssql:iscloud",
+        "container": {
+          "grid-container": [
+            {
+              "name": "%title.configurations%",
+              "row": 0,
+              "col": 0,
+              "widget": {
+                "modelview": {
+                  "id": "ml.tasks"
+                }
+              }
+            },
+            {
+              "name": "%title.tasks%",
+              "row": 0,
+              "col": 1,
+              "widget": {
+                "tasks-widget": [
+                  "mls.command.managePackages",
+                  "mls.command.odbcdriver",
+                  "mls.command.mlsdocs"
+                ]
+              }
+            }
+          ]
+        }
       }
     ]
   },
   "dependencies": {
-    "vscode-nls": "^4.0.0"
+    "vscode-nls": "^4.0.0",
+    "vscode-languageclient": "^5.3.0-next.1"
   },
   "devDependencies": {
     "@types/mocha": "^5.2.5",
     "@types/node": "^10.14.8",
-    "@types/uuid": "^3.4.5",
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.17.0",
     "mocha-multi-reporters": "^1.1.7",
     "should": "^13.2.1",
     "typemoq": "^2.1.0",
     "vscode": "1.1.26"
-  },
```

Comment:
Leave this in - it's generated by the VSCE package tool but there's no harm in having it checked in (and avoids having this be added every time someone runs the tool locally)

Reply:
I cannot install the extension when it has this data and I checked other extension. none has this data

---
In reply to: [362898526](https://github.com/microsoft/azuredatastudio/pull/8796#discussion_r362898526) [](ancestors = 362898526)

Reply:
What do you mean? It works just fine for these extensions :

https://github.com/microsoft/azuredatastudio/blob/master/extensions/admin-tool-ext-win/package.json#L84
https://github.com/microsoft/azuredatastudio/blob/master/extensions/agent/package.json#L93
https://github.com/microsoft/azuredatastudio/blob/master/extensions/profiler/package.json#L84

Are you getting an error when you try to install the extension?

Reply:
ok, not sure why I couldn't install the extension when I had this. why some extensions like notebook don't have this? what id should I use? the biggest one I found is 55. 56 then? 

---
In reply to: [363409904](https://github.com/microsoft/azuredatastudio/pull/8796#discussion_r363409904) [](ancestors = 363409904)

Reply:
I asked Alan about this and he says when I add the extension to the gallery then I would know the id and I can add this part, for now I should remove it

---
In reply to: [363465663](https://github.com/microsoft/azuredatastudio/pull/8796#discussion_r363465663) [](ancestors = 363465663,363409904)